sound/soc/sof/sof-client-probes.c

```diff
@@ -489,8 +489,8 @@ static ssize_t sof_probes_dfs_points_read(struct file *file, char __user *to,
 	struct sof_probes_priv *priv = cdev->data;
 	struct device *dev = &cdev->auxdev.dev;
 	struct sof_probe_point_desc *desc;
+	int remaining, offset;
```

Comment:
a typo in the commit message "loose"->"lose"

Reply:
fixed, thanks.